src/react/ErrorPage.jsx

```diff
@@ -28,12 +28,17 @@ class ErrorPage extends Component {
                 description="error message when an unexpected error occurs"
               />
             </p>
+            {message !== null ? (
+              <div>
```

Comment:
Could you please give this line role="note" (if you think the message is something that can be ignored) or role="alert" (if you think the message is something everyone would want to read (sounds more likely)?

Reply:
That is unless it's coming in with <formatted message> already....

Reply:
done